test/test_example.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from src.example import add_numbers, greet
+
+def test_add_numbers_positive():
```

Comment:
Add another assertion

```suggestion
def test_add_numbers_positive():
        assert add_numbers(10, 25) == 35
```